src/libraries/System.Security.Cryptography.Algorithms/ref/System.Security.Cryptography.Algorithms.cs

```diff
@@ -544,6 +544,7 @@ public abstract partial class RandomNumberGenerator : System.IDisposable
         public static void Fill(System.Span<byte> data) { }
         public abstract void GetBytes(byte[] data);
         public virtual void GetBytes(byte[] data, int offset, int count) { }
+        public static byte[] GetBytes(int byteCount) { throw null; }
```

Comment:
Looks like only the implementation got the parameter name update, missed it in the ref :)

Reply:
Huh you're right. I would have expected that to break the `ApiCompat` check but `dotnet build` doesn't complain. :shrug: .

Reply:
I believe ApiCompat only sees the type and modifiers from a signature, and they were both `public static byte[] GetBytes(System.Int32)`.